src/runtime/containerd-shim-v2/metrics.go

```diff
@@ -31,11 +31,11 @@ func marshalMetrics(ctx context.Context, s *service, containerID string) (*googl
 	return data, nil
 }
 
-func statsToMetrics(stats *vc.ContainerStats) *cgroups.Metrics {
-	metrics := &cgroups.Metrics{}
+func statsToMetrics(stats *vc.ContainerStats) *v1.Metrics {
```

Comment:
I wonder if we might want to rename these functions `statsToCGroupsV1Metrics()` or something similar?